test/configuration_test.rb

```diff
@@ -0,0 +1,42 @@
+require "webpacker_test"
+
+class ConfigurationTest < Minitest::Test
+  def test_config_path
+    config_path = File.join(File.dirname(__FILE__), "test_app/config", "webpack").to_s
```

Comment:
Perhaps use setup or before block to extract these paths out so that test only have assertions? What do you think?